### ISSUE TITLE
TypeError: fixtures/Literal/fixture.js: Cannot read property 'key' of undefined

### ISSUE THREAD

**Issue by sanemat** [twada/babel-plugin-espower#3]
Hi,

22 tests fail on master head.

```
$ npm test

> babel-plugin-espower@0.2.2 test /Users/sane/work/js-study/babel-plugin-espower
> mocha
(snip)
 1 passing (678ms)
  22 failing

  1) babel-plugin-espower Literal:
     TypeError: fixtures/Literal/fixture.js: Cannot read property 'key' of undefined
      at traverseUp (lib/babel-estree-path.js:6:33)
      at traverseUp (lib/babel-estree-path.js:18:9)
      at traverseUp (lib/babel-estree-path.js:18:9)
      at traverseUp (lib/babel-estree-path.js:18:9)
      at estreePath (lib/babel-estree-path.js:24:5)
      at enterTraversalPath (lib/create-espower-visitor.js:33:24)
      at TraversalPath.handlers.(anonymous function).enter (lib/create-espower-visitor.js:84:17)
      at Context.<anonymous> (test/test.js:13:28)

  2) babel-plugin-espower Identifier:
     TypeError: fixtures/Identifier/fixture.js: Cannot read property 'key' of undefined
      at traverseUp (lib/babel-estree-path.js:6:33)
      at traverseUp (lib/babel-estree-path.js:18:9)
      at traverseUp (lib/babel-estree-path.js:18:9)
      at traverseUp (lib/babel-estree-path.js:18:9)
      at estreePath (lib/babel-estree-path.js:24:5)
      at enterTraversalPath (lib/create-espower-visitor.js:33:24)
      at TraversalPath.handlers.(anonymous function).enter (lib/create-espower-visitor.js:84:17)
      at Context.<anonymous> (test/test.js:13:28)

(snip)
``` 

My environments.
```
$ node --version
v0.12.2
xtend@4.0.0
espurify@1.2.0
mocha@2.2.4
espower@0.11.0
babel-core@5.2.5
```

**Comment by twada** [comment]
@sanemat Thanks! I'll fix it.

**Comment by twada** [comment]
Just released [v0.3.0](https://github.com/twada/babel-plugin-espower/releases/tag/v0.3.0). Thank you for reporting!

**Comment by sanemat** [comment]
@twada Great! :heart_eyes:
